### ISSUE TITLE
Pan/Zoom events don't work as appropriate on touch-screen Windows 8

### ISSUE THREAD

**Issue by iriberri** [CartoDB/cartodb.js#313]
From SB: 4390616

**Comment by iriberri** [comment]
PS: Waiting for specific details about browsers/versions

**Comment by Xatpy** [comment]
I can zoom and pan. But I'm having problems with clicks and infowindows. 

The problem with Internet Explorer in touch mobiles (tested with Nokia Lumia 635) is that the click event is being triggered twice. So, because of that: no infowindows is appearing. 
I've tested it with the following code in `cartodb.uncompressed.js : 11427`:
```
nativeHandler = function (element, fn, args) {
        return function (event) {
          event = fixEvent(event || ((this.ownerDocument || this.document || this).parentWindow || win).event, true)
          debugger
          if ([event].concat(args)[0].type === "click") {
            alert(nativeHandler --> click!!!');
          }
          return fn.apply(element, [event].concat(args))
        }
      }
```
I check the same code with Android 4.4 (bq aquaris 4.5) and iOS7 (iPhone 4S) with Chrome and Safari and worked ('click' event triggered only once)
I've recorded a video with it--> [Link](https://dl.dropboxusercontent.com/u/22131567/ie_click.mp4) (sorry, video was uploaded flipped)

**Comment by Xatpy** [comment]
With [this PR](https://github.com/CartoDB/wax/pull/6) and [this](https://github.com/CartoDB/cartodb.js/pull/334) it should be resolved, except drag and infowindow with Internet Explorer 11 (explanation [here](https://github.com/CartoDB/wax/pull/6) )

**Comment by Xatpy** [comment]
Fixed with **3.12.6** and SupportBee replied.

Videos about the issue:
- [Infowindow](https://dl.dropboxusercontent.com/u/22131567/VID_20150223_124757.mp4)
- [Pan / Zoom](https://dl.dropboxusercontent.com/u/22131567/VID_20150223_175931.mp4)
